README.rst

```diff
@@ -28,6 +28,10 @@ You can install all required packages using `poetry`__ with::
    .. __: http://www.phrap.org/phredphrapconsed.html
    .. __: https://www.girinst.org/repbase/
 
+To use the packages installed via poetry, you have to activate the poetry 
+environment via::
+
+    poetry shell
```

Comment:
Can you add here something like
```
or run your commands with::

  poetry run <your command>
```
for completeness?

Reply:
Done